src/bots/pause_bot.py

```diff
@@ -0,0 +1,196 @@
+            'value': self.blocks_till_pause_is_valid,
+        })
+
+        self.pause_prefix = contracts.deposit_security_module.functions.PAUSE_MESSAGE_PREFIX().call()
+        logger.info({'msg': f'Call `PAUSE_MESSAGE_PREFIX()`.', 'value': self.pause_prefix})
+
+        transports = []
+
+        if TransportType.RABBIT in variables.MESSAGE_TRANSPORTS:
+            transports.append(RabbitProvider(
+                client='pauser',
+                routing_keys=[MessageType.PING, MessageType.PAUSE],
+                message_schema=Schema(Or(PauseMessageSchema, PingMessageSchema)),
+            ))
+
+        if TransportType.KAFKA in variables.MESSAGE_TRANSPORTS:
+            transports.append(KafkaMessageProvider(
+                client=f'{variables.KAFKA_GROUP_PREFIX}pause',
+                message_schema=PauseMessageSchema,
+            ))
+
+        if not transports:
+            logger.error({'msg': 'No transports found', 'value': variables.MESSAGE_TRANSPORTS})
+            raise ValueError(f'No transports found. Provided value: {variables.MESSAGE_TRANSPORTS}')
+
+        self.message_storage = MessageStorage(
+            transports,
+            filters=[
+                message_metrics,
+                get_pause_messages_sign_filter(self.pause_prefix),
+            ],
+        )
+
+        BUILD_INFO.labels(
+            'Pauser bot',
+            variables.NETWORK,
+            None,
+            None,
+            None,
+            None,
+            None,
+            None,
+            None,
+            None,
+            variables.KAFKA_TOPIC,
+            variables.ACCOUNT.address if variables.ACCOUNT else '0x0',
+            variables.CREATE_TRANSACTIONS,
+        )
+
+    def run_as_daemon(self):
+        """Super-Mega infinity cycle!"""
+        while True:
+            self._waiting_for_new_block_and_run_cycle()
+
+    @timeout_decorator.timeout(variables.MAX_CYCLE_LIFETIME_IN_SECONDS)
+    def _waiting_for_new_block_and_run_cycle(self):
+        try:
+            self.run_cycle()
+        except timeout_decorator.TimeoutError as exception:
+            # Bot is stuck. Drop bot and restart using Docker service
+            logger.error({'msg': 'Pauser bot do not respond.', 'error': str(exception)})
+            raise timeout_decorator.TimeoutError('Pauser bot stuck. Restarting using docker service.') from exception
+        except BlockNotFound as error:
+            logger.warning({'msg': 'Fetch block exception (BlockNotFound)', 'error': str(error)})
+            time.sleep(15)
+        except NoActiveProviderError as exception:
+            logger.error({'msg': 'No active node available.', 'error': str(exception)})
+            raise NoActiveProviderError from exception
+        except Exception as exception:
+            logger.warning({'msg': 'Unexpected exception.', 'error': str(exception)})
+            time.sleep(15)
+        else:
+            time.sleep(15)
```

Comment:
@F4ever @Eddort 
is there a pattern to write "time.sleep(15)" in else condition? Maybe it'd be better to place it after self.run_cycle() calling? 

Ps: I dont't know about common dealings in python.